ethportal-api/Cargo.toml

```diff
@@ -11,15 +11,47 @@ categories = ["cryptography::cryptocurrencies"]
 authors = ["https://github.com/ethereum/trin/graphs/contributors"]
 
 [dependencies]
+anyhow = "1.0.68"
```

Comment:
It seems like this pr is a good candidate to run [udeps](https://crates.io/crates/cargo-udeps) locally if you haven't already